components/brave_sync/brave_sync_service_unittest.cc

```diff
@@ -826,6 +826,20 @@ TEST_F(BraveSyncServiceTest, OnSyncReadyNewToSync) {
   EXPECT_TRUE(profile()->GetPrefs()->GetBoolean(syncer::prefs::kSyncBookmarks));
 }
 
+TEST_F(BraveSyncServiceTest, OnSyncReadyModelNotYetLoaded) {
+  EXPECT_NE(sync_service()->model_, nullptr);
```

Comment:
I think we have better way to test if model is truly loaded not just nullptr
see `components/bookmarks/test/test_bookmark_client.cc`, we can have our own create method but not calling `DoneLoading` and then after calling `OnSyncReady`, we call `DoneLoading`

Reply:
cool, thanks for the advise, will redo

Reply:
redone, though I had to add patch  to access private `BookmarkModel::DoneLoading`.
cc @bridiver